coffee_github/src/repository.rs

```diff
@@ -221,9 +219,7 @@ impl Repository for Github {
                 } else {
                     "main".to_owned()
                 };
-                let (git_head, date_of_the_last_commit) = self.git_info().await?;
-                self.git_head = Some(git_head);
-                self.date_of_the_last_commit = Some(date_of_the_last_commit);
+                self.git_info().await?;
```

Comment:
You do not need to have the method of the object, but the procedure can be static (without self) or just a macros, and this get_info method can return the information about the commit header and the commit time